README.md

```diff
@@ -955,7 +955,13 @@ IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2022. [PDF](https://arxiv.org
 IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2022. [Project page](https://sites.google.com/view/evsnn), [PDF](https://openaccess.thecvf.com/content/CVPR2022/papers/Zhu_Event-Based_Video_Reconstruction_via_Potential-Assisted_Spiking_Neural_Network_CVPR_2022_paper.pdf), [Suppl. Mat.](https://openaccess.thecvf.com/content/CVPR2022/supplemental/Zhu_Event-Based_Video_Reconstruction_CVPR_2022_supplemental.pdf), [Code](https://github.com/LinZhu111/EVSNN).
 - <a name="Liu23tpami"></a>Liu, S., Dragotti, P.L.,    
 *[Sensing Diversity and Sparsity Models for Event Generation and Video Reconstruction from Events](https://doi.org/10.1109/TPAMI.2023.3278940)*,    
-IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2023. [Code](https://github.com/pld-group/V2E2V)
+IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2023. [Code](https://github.com/pld-group/V2E2V), [Code for data generation](https://github.com/lsying009/V2E_Generation)
+    - <a name="Liu22icassp"></a>Liu, S., Alexandru, R., Dragotti, P.L., 
+    *[Convolutional ISTA Network with Temporal Consistency Constraints for Video Reconstruction from Event Cameras](https://ieeexplore.ieee.org/abstract/document/9746331)*,    
+    IEEE Int. Conf. Acoust., Speech, Signal Proc. (ICASSP), 2022. [Code](https://github.com/pld-group/V2E2V)
+    - <a name="Liu24arxiv"></a>Liu, S., Dragotti, P.L.,    
+    *[Enhanced Event-Based Video Reconstruction with Motion Compensation](https://doi.org/10.1109/TPAMI.2023.3278940)*,    
```

Comment:
this TPAMI link does not correspond to the title of this publication. Please check

Reply:
I think it should be this link: https://arxiv.org/abs/2403.11961